Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/PANOSPolicyOptimizer.yml

```diff
@@ -73,6 +76,9 @@ script:
       predefined:
       - pre
       - post
+    - description: Is there an alternate device group to query?
+      name: override_device_group
+      type: textArea
```

Comment:
The type `textArea` is for large inputs. You can skip this configuration key in your case.